package.json

```diff
@@ -429,4 +435,4 @@
     "which": "^2.0.1",
     "yauzl": "^2.10.0"
   }
-}
+}
```

Comment:
Unintended change?

Reply:
vscode 1.64.2 json  formatter trim the tail \n

Reply:
Colour me surprised

Reply:
In that case, thank you for correcting it :)